maxim/models/dataset.py

```
@@ -9,128 +9,215 @@ class VariableType(str):
-
-    def to_json(self):
+    type: Literal["text", "json", "file"]
+    payload: Union[str, Dict[str, Any], List[Attachment]]
+    
+    def to_json(self) -> Dict[str, Any]:
+        """Convert the Variable to a JSON-serializable dictionary."""
         return {"type": self.type, "payload": self.payload}
-
+    
     @classmethod
     def from_json(cls, data: Dict[str, Any]) -> "Variable":
```

Ruff (0.12.2)

`from_json` is too complex (11 > 10)

(C901)